deployment/helm/dnsmasqaks/Chart.yaml

```diff
@@ -16,7 +16,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.1.0
+version: 0.5.0
```

Comment:
This was done (all of them), due to several tests in my own main branch and run the workflow. I think you are right, I will move it back to just one version higher instead of skipping a few.

Reply:
Done